corehq/apps/hqadmin/reports.py

```diff
@@ -1166,3 +1168,70 @@ def _create_row(self, log, *args, **kwargs):
         row = super(DeviceLogSoftAssertReport, self)._create_row(log, *args, **kwargs)
         row.append(log.domain)
         return row
+
+
+class CommCareVersionReport(AdminReport, ElasticTabularReport):
+    slug = "commcare_version"
+    name = ugettext_lazy("CommCare Version")
+
+    @property
+    def headers(self):
+        versions = get_all_versions()
+        headers = DataTablesHeader(
+            DataTablesColumn(_("Project"))
+        )
+        for version in versions:
+            headers.add_column(DataTablesColumn(version))
+        return headers
+
+    @property
+    def rows(self):
+        versions = get_all_versions()
+
+        def get_query(version):
+            return {
```

Comment:
Instead of returning json, could you use our ES ORM?

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/forms.py
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/aggregations.py